MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -324,6 +325,11 @@ public RasterizerState RasterizerState
             }
         }
 
+        /// <summary>
+        /// Blend factor for alpha blending.
+        /// </summary>
+        public Color BlendFactor { get { return _blendFactor; } set { _blendFactor = value; } }

```

Comment:
This won't work.

You can change the blend factor here yet the blend state won't reapply until `_blendStateDirty` is set to true.